go/host/host.go

```diff
@@ -356,6 +356,20 @@ func (h *host) startProcessing() {
 	i := int32(0)
 	roundInterrupt := &i
 
+	if h.config.NodeType == common.Sequencer {
+		go func() {
+			for {
```

Comment:
Nice! Love this. But this goroutine will get leaked, need to hook it into the host stop mechanisms. Also probably cleaner to put it in a function so it's `go startRollupPublisher()` or something.